gatsby-config.js

```diff
@@ -80,6 +96,7 @@ module.exports = {
       resolve: 'gatsby-plugin-netlify-cms',
       options: {
         modulePath: `${__dirname}/src/cms/cms.ts`,
+        manualInit: true,
```

Comment:
This is for the configuration to accept an manual init. https://www.gatsbyjs.org/packages/gatsby-plugin-netlify-cms/#manualinit